curve25519-dalek/src/constants.rs

```diff
@@ -94,6 +78,9 @@ pub const BASEPOINT_ORDER: Scalar = Scalar {
     ],
 };
 
+#[allow(deprecated)]
+pub(crate) const BASEPOINT_ORDER_PRIVATE: Scalar = BASEPOINT_ORDER;
```

Comment:
Maybe reverse this relationship so `BASEPOINT_ORDER_PRIVATE` defines the constant and `BASEPOINT_ORDER` points to `BASEPOINT_ORDER_PRIVATE`?

Reply:
ah ofcourse, thanks :purple_heart: